src/test-jdk17/java/com/fasterxml/jackson/dataformat/xml/records/failing/XmlWrapperRecord517Test.java

```diff
@@ -0,0 +1,59 @@
+package com.fasterxml.jackson.dataformat.xml.records.failing;
+
+import java.util.List;
+
+import com.fasterxml.jackson.dataformat.xml.XmlTestBase;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+import org.junit.Test;
```

Comment:
Should be imported after `java.util.List` as per coding style.

Reply:
Done thank you!